### ISSUE TITLE
Could you please share the hyper-parameters for KUKA use PPO?

### ISSUE THREAD

**Issue by LizstCat** [ikostrikov/pytorch-a2c-ppo-acktr#64]
I have tried both Discrete and Continous action space in Kuka by set the Kuka env parameter IsDiscrete true or false, but I still can not make it work. Could you please tell me the parameters you used for Kuka?
Thanks very much!!
ps. I use pybullet 2.87 with pybullet 1.92, which has a reward​ based​​ on ​the​ object​ ​reaching​ ​a ​certain​ ​height​ ​ (main)​ ​and distance​ ​between​ ​gripper​ ​and​ ​object​(minor​ ​ reward​ ​ shaping, every step). Thanks again!

**Comment by ikostrikov** [comment]
Did you install pybullet from sources?

**Comment by ikostrikov** [comment]
I think that standard hyperparameters should work. It's just a very difficult task, so it might take a lot of updates to solve it.
